### ISSUE TITLE
Significant struggles with name identification

### ISSUE THREAD

**Issue by BenSturgeon** [OmkarPathak/pyresparser#31]
Thank you very much for the work you've done on this. 

While the results of this are currently fairly good I've noticed names are a big struggle. I even ran your resume as a sample through the system and it returned "www.omkarpathak.in" for that field. 

Do you think adding negative patterns for it to check against is the smartest short term solution for this problem? Otherwise do you think more training is required on the part of the NLP model regarding names?

If you need access to more data I have access to a large amount of CVs which I'd be happy to share.

Thanks again for your continued work on this project.

**Comment by OmkarPathak** [comment]
@BenSturgeon yes. We need a large dataset of resumes to train model to produce more robust results. If you can share the CVs it would be really helpful 😄

**Comment by BenSturgeon** [comment]
@OmkarPathak  Awesome, I'll send you an email with the google drive link containing a large amount of CVs. 

I'd be happy to contribute by helping with labeling as well if you'd be interested in sharing the process with me.

**Comment by OmkarPathak** [comment]
@BenSturgeon would be happy to share.

**Comment by aditya-malte** [comment]
Hi,
So do we now have a large dataset? Would be great if it was open-source
